process/headerCheck/headerChecker.go

```diff
@@ -0,0 +1,210 @@
+package headerCheck
```

Comment:
rename file. it is a headerSignatureVerify

Reply:
done.